### ISSUE TITLE
Pricing for spelled items.

### ISSUE THREAD

**Issue by Sunjarfishy** [TF2Autobot/tf2autobot#649]
Add a way to have separate sell listings for spelled variants of the same item

**Comment by joekiller** [comment]
Seems like this would effectively be adding/finishing spell skus right? I might be willing to take this next.

Can another member confirm the work needed here. I'll post again if I get to look closer sooner.

**Comment by idinium96** [comment]
Yes, it's pretty much the same as how I implement support for painted items, but keep in mind that you can have more than one spell on an item.

**Comment by joekiller** [comment]
Relevant: https://github.com/TF2Autobot/tf2autobot/commit/a8662706034b4d236ae6fbab0549a8e02b97cbdc
